### DISCUSSION TITLE
Energy Forecast

### DISCUSSION THREAD

**Discussion by FSHelgeland** [ekutner/home-connect-hass#241] [General]
My dishwasher gives me am energy forecast at startup. What does that tell me, now as I am writing this I got 77% from the sensor energy forecast. Would have loved to make an energy sensor off it, and add it to the energy dashboard. But in order to do so I need to know what that sensor actually gives me, anyone who knows? 

Br
Freddy

**Comment by LasseTheDude** [comment]
Bump 😇
